RecoTauTag/Configuration/python/boostedHPSPFTaus_cfi.py

```diff
@@ -1,13 +1,21 @@
 import FWCore.ParameterSet.Config as cms
+import PhysicsTools.PatAlgos.tools.helpers as configtools
 
 def addBoostedTaus(process):
     from PhysicsTools.PatAlgos.tools.helpers import cloneProcessingSnippet
     from PhysicsTools.PatAlgos.tools.helpers import massSearchReplaceAnyInputTag
+
     process.load("RecoTauTag.Configuration.boostedHPSPFTaus_cff")
+    patAlgosToolsTask = configtools.getPatAlgosToolsTask(process)
+    patAlgosToolsTask.add(process.pfPileUpForBoostedTaus)
+    patAlgosToolsTask.add(process.pfNoPileUpForBoostedTaus)
+    patAlgosToolsTask.add(process.ca8PFJetsCHSprunedForBoostedTaus)
+    patAlgosToolsTask.add(process.boostedTauSeeds)
```

Comment:
having a boostedHPSPFTausTask in boostedHPSPFTaus_cff would make it simpler

... there was some lack of attention in review in the past that let this  xyz_cfi  depend on xyz_cff